_includes/serial_dump.html

```diff
@@ -0,0 +1,8 @@
+
+{% if {{include.url}} %}
+    <a class="no_icon" target="_blank" rel="noopener" href="{{include.url}}">
+{% endif %}
+
```

Comment:
```suggestion
<details>
    <summary>{{include.title}}</summary>
```

Reply:
I've modified but then need to include a code like template I'm not sure this is how you wanted it.

Reply:
sorry, I thought it was clear, I wanted to make this file appear via spoiler by pressing the appropriate button